e-learning-webapps/src/main/webapp/skin/less/elearning.less

```diff
@@ -1,8 +1,10 @@
 @import "./variables.less";
 #vuetify_webpack_sample {
   background: transparent;
-
   .controls {
     position: relative;
   }
-}
\ No newline at end of file
+}
+#btn-next {
+  background-color: #9e2929;
```

Comment:
Please use eXo color variables from this file https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/variables.less